cyber/record/record_viewer.h

```diff
@@ -31,25 +31,68 @@ namespace apollo {
 namespace cyber {
 namespace record {
 
-class RecordMessage;
-
+/**
+* @brief The record viewer.
+*/
 class RecordViewer {
  public:
   using RecordReaderPtr = std::shared_ptr<RecordReader>;
 
+  /**
+  * @brief The constructor with single reader.
+  *
+  * @param reader
+  * @param begin_time
+  * @param end_time
+  * @param channels
+  */
   RecordViewer(const RecordReaderPtr& reader, uint64_t begin_time = 0,
                uint64_t end_time = UINT64_MAX,
                const std::set<std::string>& channels = std::set<std::string>());
+
+  /**
+  * @brief The constructor with multiple readers.
+  *
+  * @param readers
+  * @param begin_time
+  * @param end_time
+  * @param channels
+  */
   RecordViewer(const std::vector<RecordReaderPtr>& readers,
                uint64_t begin_time = 0, uint64_t end_time = UINT64_MAX,
                const std::set<std::string>& channels = std::set<std::string>());
 
+  /**
+  * @brief Is this record reader is valid.
+  *
+  * @return True for valid, false for not.
+  */
   bool IsValid() const;
-  bool Update(RecordMessage* message);
-  uint64_t begin_time() const;
-  uint64_t end_time() const;
-  std::set<std::string> GetChannelList() const;
```

Comment:
the change can break existing callers.  have we checked the existing references and modify them accordingly

Reply:
Fixed in new request.